cmd/server/main.go

```diff
@@ -81,6 +80,20 @@ func checkAndLog(err error) {
 	}
 }
 
+type Config struct {
+	// open, community, restricted
+	privacyMode roomdb.PrivacyMode
+}
+
+// cblgh: i dont get why we need a context here, if it is potentially a bad code smell?
```

Comment:
We can ignore it here (you can use `_ context.Context`) but the database code later will need it and changing interfaces later is just hassle no-one needs.

Reply:
ah yeah, i was kind of guessing at that (but i have no knowledge yet about the database stuff—i guess i will have that though when this is merged! exciting~~~)